rasa/core/schemas/domain.yml

```
@@ -31,3 +31,15 @@ mapping:
     sequence:
     - type: "str"
     allowempty: True
+  config:
+    type: "map"
+    allowempty: True
+    mapping:
+      store_entities_as_slots:
+        type: "bool"
+      session_length:
+        type: "number"
+        range:
+          min: 0
```

```suggestion
          min-ex: 0
```
since we want too exclude 0 (https://pykwalify.readthedocs.io/en/unstable/validation-rules.html#range)

---

I thought we'd use `0` for no sessions? We should probably also handle that :-D

---

oh yes totally - never mind my comment and yes we need to handle the no-sessions behaviour (probably in `processor._update_tracker_session()`)

---

handled it in `_has_session_expired` + tested